g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/CategoriesEnum.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.common.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+
+public enum CategoriesEnum {
+    DATES(0, "dates"),
+    NUMBERS(1, "numbers"),
+    PLURALS(2, "plurals"),
+    MEASUREMENTS(3, "measurements"),
+    CURRENCIES(4, "currencies"),
+    DATE_FIELDS(5, "dateFields");
+
```

Comment:
I suggest that use the bit value to express the DATES, NUMBERS ... such as 
    DATES(1, "dates"),
    NUMBERS(2, "numbers"),
    PLURALS(4, "plurals"),
    MEASUREMENTS(8, "measurements"),
    CURRENCIES(16, "currencies"),
    DATE_FIELDS(32, "dateFields");

Reply:
fixed